python/test/segy.py

```diff
@@ -572,6 +572,12 @@ def test_open_fails_unstructured(self):
             self.assertListEqual(list(f.attributes(189)[:]),
                                  [(i // 5) + 1 for i in range(len(f.trace))])
 
+    def test_traceaccess_from_array(self):
+        a = np.zeros(10, dtype = np.int)
```

Comment:
can you test for `np.int32` and `np.int64`?  Then this looks very good and ready to merge.